src/bot.js

```diff
@@ -12,8 +14,11 @@ const FOOTER =
     'El calendario de eventos completo lo podés mirar en http://meetupjs.com.ar/calendario.html'
 const HODOR_FOOTER =
     'Hodor hodor hodor hodor hodor hodor hodor hodor hodor http://meetupjs.com.ar/calendario.html'
+const BIRTHDAY_FOOTER =
+    'Si querés que la comunidad te salude por tu cumpleaños, completá el siguiente formulario: https://docs.google.com/forms/d/e/1FAIpQLSdV9nHG84MxpvM9ewHoIUpnCHspGqcoSealCrq8ajqsAhAhWQ/viewform?usp=sf_link'
```

Comment:
El mensaje sugiere que vos agregues tu propio cumple, le cambiaría el enfoque como para que cualquiera agregue cualquier cumpleaños.

Puse `saludito` porque me pintó jaja Le saqué el queryParam del final también.

```suggestion
    '> Para agregar un saludito usá este formulario https://docs.google.com/forms/d/e/1FAIpQLSdV9nHG84MxpvM9ewHoIUpnCHspGqcoSealCrq8ajqsAhAhWQ/viewform'
```